package.json

```diff
@@ -10,8 +10,8 @@
     "build:serve": "docusaurus build && serve build/"
   },
   "dependencies": {
-    "@docusaurus/core": "^2.0.0-alpha.64",
-    "@docusaurus/preset-classic": "^2.0.0-alpha.64",
+    "@docusaurus/core": "^2.0.0-beta.ff31de0ff",
```

Comment:
Hey @metaverde. Thanks for your first PR! 🥳 
This update to docusaurus is breaking the build. One file `docs/how-does-it-work/xp.md` has an `image` property in the frontmatter that seems to be invalid in the beta version of Docusaurus. If we revert Docusaurus to the previous version, the build succeeds. :) Was there a reason for the bump in versions? Going forwards, we will be migrating to beta.0 as part of  a wiki overhaul that @jonathanprozzi has started working on. 

It is good practice to run `yarn build` on your local build to check your updates before you push & make a PR. That way you know your work will deploy without issue. This is especially important if you're upgrading any of the existing packages for a project. 😄

Reply:
The version bump was likely due to us walking through installing node/yarn/npm through the push process. We'll get it fixed and whipped into shape 👍